subtests/docker_cli/systemd_in_container/systemd_in_container.py

```diff
@@ -0,0 +1,50 @@
+r"""
```

Comment:
I'm fine with either way as well. Whatever is easier for you guys to review. I don't hide the fact that this is edited copy of `subtests/docker_cli/selinux_labels/` so I left it next to it for a start.